torch/csrc/jit/runtime/profiling_record.cpp

```diff
@@ -1,12 +1,34 @@
 #include <torch/csrc/jit/runtime/profiling_record.h>
-#include <torch/csrc/jit/passes/clear_profiling.h>
-#include <torch/csrc/jit/passes/constant_propagation.h>
+#include <torch/csrc/jit/jit_log.h>
 #include <torch/csrc/jit/runtime/graph_executor.h>
 #include <torch/csrc/jit/runtime/interpreter.h>
+#include <ostream>
 
 namespace torch {
 namespace jit {
 
+c10::ShapeSymbol ShapeSymbolTable::toSymbol(
```

Comment:
I don't understand the name `toSymbol`, `getOrCreateSymbolForDim`? Shouldn't this be a method on whatever owns dims2symbols? It is weird that this is taking the map as an argument.